flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/stream/MiscSemanticTests.java

```diff
@@ -31,6 +32,7 @@ public class MiscSemanticTests extends SemanticTestBase {
     public List<TableTestProgram> programs() {
         return List.of(
                 WindowRankTestPrograms.WINDOW_RANK_HOP_TVF_NAMED_MIN_TOP_1,
-                CalcTestPrograms.CURRENT_WATERMARK);
+                CalcTestPrograms.CURRENT_WATERMARK,
+                JoinTestPrograms.MULTI_JOIN_BUG);
```

Comment:
why put in `Misc`? Why not some new `JoinSemanticTests`?

Reply:
Moved to `JoinSemanticTests`